presto-main/src/test/java/com/facebook/presto/operator/scalar/sql/TestArrayArithmeticFunctions.java

```diff
@@ -65,4 +74,45 @@ public void testArrayAverage()
         assertFunction("array_average(array[null, null])", DOUBLE, null);
         assertFunction("array_average(null)", DOUBLE, null);
     }
+
+    @Test
+    public void testArrayFrequencyBigint()
+    {
+        FunctionAndTypeManager functionAndTypeManager = createTestFunctionAndTypeManager();
+        MapType type = new MapType(BIGINT,
+                INTEGER,
+                methodHandle(TestRowType.class, "throwUnsupportedOperation"),
+                methodHandle(TestRowType.class, "throwUnsupportedOperation"));
+        TypeSignature typeSignature = TypeSignature.parseTypeSignature(type.getDisplayName());
+
+        assertFunction("array_frequency(cast(null as array(Bigint)))", functionAndTypeManager.getType(typeSignature), null);
+        assertFunction("array_frequency(array[cast(null as Bigint), cast(null as Bigint), cast(null as Bigint)])", functionAndTypeManager.getType(typeSignature), null);
```

Comment:
Hmm, while this behavior is sort of reasonable, I wonder whether this is expected. @kaikalur  what do you think should be the right outcome if the input array only has null elements?

Reply:
This seems reasonable (especially because we can't represent null as a key in the result map)

Reply:
I changed the code to ignore null elements (can deal with arrays that have both null and non-null elements), so array of nulls now returns an empty map.

Reply:
Make sure to document this null behavior